support-frontend/assets/helpers/user/user.js

```diff
@@ -82,7 +82,11 @@ const init = (dispatch: Function, actions: UserSetStateActions = defaultUserActi
 
   const emailFromBrowser = getEmailFromBrowser();
 
-  if (isTestUser()) {
+  /*
+   * Prevent use of the test stripe token if user us logged in, as support-workers will try it in live mode
+   * if the identity cookie is present
+   */
+  if (isTestUser() && !userAppearsLoggedIn) {
```

Comment:
great, but is it easy to check if the user appears logged in as a test user? if so might be worth making that additional check to avoid the opposite problem.

Reply:
It would be simple to also check if the user's email matches the test username, i.e.:
`window.guardian.email.startsWith(cookie.get('_test_username'))`
Would this help?

Reply:
I've pushed a commit which I think solves this

Reply:
nice work, that's a good idea. Now we would only see a problem if it was a valid test user but had expired (48H), or someone was using an incognito window to sign up as a test user, which all seems unlikely!